tailcfg/tailcfg.go

```diff
@@ -1940,6 +1943,10 @@ type SSHAction struct {
 	// AllowLocalPortForwarding, if true, allows accepted connections
 	// to use local port forwarding if requested.
 	AllowLocalPortForwarding bool `json:"allowLocalPortForwarding,omitempty"`
+
+	// SessionHauling, if non-empty, is the IP address of a peer to stream this
+	// SSH session's logs to.
+	SessionHauling string `json:"sessionHauling,omitempty"`
```

Comment:
should this be the peers NodeID instead of the IP?
If it has to be an IP then it should likely be `netip.Addr`